cmd/syncthing/gui.go

```diff
@@ -503,6 +503,7 @@ func restGetSystem(w http.ResponseWriter, r *http.Request) {
 	cpuUsageLock.RUnlock()
 	res["cpuPercent"] = cpusum / float64(len(cpuUsagePercent)) / float64(runtime.NumCPU())
 	res["pathSeparator"] = string(filepath.Separator)
+	res["uptime"] = int32(time.Since(startTime).Seconds())
```

Comment:
This could be just a regular `int` as we don't specifically require it to be 32 bits.